tasks.yaml

```diff
@@ -42,12 +42,12 @@ tasks:
           # Test zarf init due to containerd issue - https://github.com/defenseunicorns/zarf/issues/592
           uds zarf init --confirm --no-progress
 
-  - name: build-image
-    actions:
-      - description: Build the custom k3s image
-        cmd: docker build -t ${IMAGE_NAME}:${VERSION} --build-arg K3S_TAG=${VERSION} docker/
+  # - name: build-image
+  #   actions:
+  #     - description: Build the custom k3s image
+  #       cmd: docker build -t ${IMAGE_NAME}:${VERSION} --build-arg K3S_TAG=${VERSION} docker/
 
-  - name: publish-image 
-    actions:
-      - description: Publish the custom k3s image
-        cmd: docker buildx build --push --platform linux/arm64/v8,linux/amd64 --tag ${IMAGE_NAME}:${VERSION} docker
+  # - name: publish-image
+  #   actions:
+  #     - description: Publish the custom k3s image
+  #       cmd: docker buildx build --push --platform linux/arm64/v8,linux/amd64 --tag ${IMAGE_NAME}:${VERSION} docker
```

Comment:
@mjnagel - is it safe to assume that these tasks are kept around as comments in the event that we need to build/publish our own k3s image again in the future?

Reply:
Yeah, noted in the description:
> Comment out/hide workflows for custom image building - these could be fully removed but they may be useful in the future for building custom gpu images, etc

Primarily keeping because it seemed like a nice framework for building multi-version custom k3s images. While I would hope we don't have any similar issues like the containerd bug, there may be cases for custom images with more functionality built in (like gpu support).